artemis-server/src/main/java/org/apache/activemq/artemis/core/persistence/impl/journal/JournalStorageManager.java

```diff
@@ -797,6 +764,14 @@ private Map<Long, Pair<String, Long>> recoverPendingLargeMessages() throws Excep
       return largeMessages;
    }
 
+   @Override
+   public void recoverLargeMessagesOnFolder(Set<Long> files) throws Exception {
+      List<String> filenames = largeMessagesFactory.listFiles("msg");
+      filenames.forEach(f -> {
+         files.add(getLargeMessageIdFromFilename(f));
+      });
+   }
```

Comment:
Might be clearer to call the Set 'messageIDs' or something, given it doesnt really contain files/names in the end.

Reply:
At this leve I don't have the messages yet.. just the large messages that are still on the message-folder. At this point I really have a File Name that I would need to parse to recover the messageID,.

Reply:
The List of filenames contains filenames, but the Set does not, it contains the results added to it which are messageID, so it seems weird to call the Set 'files'.

The caller calls the same Set "storedLargeMessages" which seems nicer.